src/accounts/models.py

```diff
@@ -134,13 +134,18 @@ def is_token_valid(self):
         return (self.created_at + timedelta(minutes=settings.PASS_RESET_TOKEN_TTL)) > timezone.now()
 
 
-class SHA512Token(models.Model):
-    token = models.CharField(max_length=128)
-    account = models.ForeignKey(Account, related_name="sha512_tokens", on_delete=models.CASCADE)
+class ConnectionChallenge(models.Model):
+    connection_challenge = models.CharField(
+        primary_key=True, 
+        max_length=128,
+        help_text="""A random challenge string that is independently calculated by both the client and server.
+        Traditionally, this is calculated as SHA-512 of a the Server's public key + a random shared secret generated by the client."""
```

Comment:
```suggestion
        Traditionally, this is calculated as SHA-512 of the Server's public key + a random shared secret generated by the client."""
```
I guess it was a typo

Reply:
😅